sdk/core/System.ClientModel/src/Options/ClientPipelineOptions.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ClientModel.Internal;
+
+namespace System.ClientModel.Primitives;
+
+/// <summary>
+/// Controls the creation of a pipeline used by a service client.
+/// Service clients must create a client-specific subtype of this class
+/// to pass to their constructors to allow for service-specific options
+/// with a client-wide scope.
+/// </summary>
+public class ClientPipelineOptions
+{
+    private static readonly ClientPipelineOptions _defaultOptions = new();
+    internal static ClientPipelineOptions Default => _defaultOptions;
+
+    #region Pipeline creation: Overrides of default pipeline policies
+
+    public PipelinePolicy? RetryPolicy { get; set; }
```

Comment:
Do we need ref docs for these?

Reply:
We will for sure.  Tracking refdocs with https://github.com/Azure/azure-sdk-for-net/issues/41044 to let the APIs settle before investing in those.